Util/path.py

```diff
@@ -9,16 +9,16 @@
 
 class Path(collections.MutableSequence):
 
-    def __init__(self, start=Position(),  end=Position()):
-        self.points = [start, end]
+    def __init__(self, start=None, target=None):
+        self.points = [start, target]
 
     def filter(self, threshold):
         if len(self) > 2:
-            new_points = [self.start]
-            for p1, p2 in zip(self[1:], self[2:]):
+            new_points = []
+            for p1, p2 in zip(self, self[1:]):
                 if (p1 - p2).norm >= threshold:
                     new_points.append(p1)
-            new_points.append(self.goal)
+            new_points.append(self.target)
```

Comment:
On ne risque pas de perdre le point de départ?
Ce que l'on veut est enlevé les points in entre le start et target.

Reply:
Non, il va être deplacer au plus d'un treshold. L'important c'est de conserver la target. Anyway, l'engine va recalculer le point de depart pour mettre le robot.

Reply:
Tu es sur que l'on fait pas assumption partout que le premier point égale la position du robot?

Reply:
Je vais y penser :P mais ca m'avais causer problème

Reply:
Ca donne ca si on conserve toujours le start:
```
[ (0,0), (10,0), (100,10), (100, 100)] -> [ (10,0), (100,10), (100, 100)]
[ (0,0), (10,0), (100,10), (100, 100)] -> [ (0,0), (10,0), (100,10), (100, 100)]
```